actors/builtin/miner/miner_actor.go

```diff
@@ -2074,6 +2074,7 @@ type ReplicaUpdate struct {
 	Partition          uint64
 	NewSealedSectorCID cid.Cid `checked:"true"`
 	Deals              []abi.DealID
+	UpdateProofType    abi.RegisteredUpdateProof
```

Comment:
Right now this seems to be ignored since we send the sealProofInfo.RegisteredUpdateProof type into the runtime verification call.  This is ok for now.  Slight preference to enforce that this correctly matches sealProofInfo.RegisteredUpdateProof when doing the validation step

Reply:
Unless I'm misunderstanding you, that validation happens on line 2250.

Reply:
Merging for now.